installer/manifests/keptn/values.yaml

```diff
@@ -67,6 +67,7 @@ nats:
     enabled: true
     replicas: 3
     name: nats
+    automountServiceAccountToken: false
```

Comment:
are you sure this belongs under the cluster key?

Reply:
good catch, fixed